yandex_music/artist/artist.py

```diff
@@ -284,7 +300,7 @@ async def get_albums_async(
         await client.artists_direct_albums(artist.id, page, page_size, sort_by, *args, **kwargs)
```

Comment:
should we update docs here? `artist.id_required` instead of `artist.id`

Reply:
Oh yeah, right. I completely forgot sorry. Will push new changes in a sec 🤦